jsk_pr2_robot/jsk_pr2_startup/jsk_pr2.rosinstall

```diff
@@ -1,64 +1,132 @@
 - git:
```

Comment:
please add comment, we need to install from source, because pr2 is kinetic??

Reply:
Yes, mainly because pr2 is indigo, we need to install them from source.
I update `.rosinstall` file and add comments.
We use this `.rosinstall` for nightly build in `pr1040` as `fetch`.
we need https://github.com/jsk-ros-pkg/jsk_robot/pull/1630 for nightly workspace updates